src/Security/Authentication/Twitter/src/TwitterHandler.cs

```diff
@@ -318,5 +326,79 @@ private string ComputeSignature(string consumerSecret, string tokenSecret, strin
                 return Convert.ToBase64String(hash);
             }
         }
+
+        private async Task EnsureTwitterRequestSuccess(HttpResponseMessage response)
+        {
+            var contentTypeIsJson = Equals(response.Content.Headers.ContentType, new MediaTypeHeaderValue("application/json"));
```

Comment:
Is this Object.Equals? That's brittle if they ever include parameters like a charset.

Try this:
```suggestion
            var contentTypeIsJson = string.Equals(response.Content.Headers.ContentType?.MediaType ?? "", "application/json", StringComparison.OrdinalIgnoreCase);
```